style.css

```diff
@@ -135,11 +144,230 @@ body {
   opacity: .5;
 }
 
-.main__image {
-  width: 770px;
-  height: 657px;
-}
-
 .main__title-text_highlight {
   background: #ecf3ff;
 }
+
+                   /* Мобильные устройства */
+
+@media (max-width: 768px) {
+  .menu {
+    display: none;
+ }
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/new_project2.0/pullRequest?prid=6297664)